api/src/functionalTest/java/uk/gov/hmcts/payment/functional/OnlineCardPaymentFunctionalTest.java

```diff
@@ -326,6 +322,35 @@ public void retrieveCMCCardPaymentTestShouldReturnAutoApportionedFees() {
         }));
     }
 
+    // TO BE IMPLEMENTED IN THE CARD PAYMENT SCOPE
```

Comment:
Why is this test commented.

Reply:
This test was in RefundRequestorPBAFunctionalTets class and had an @ignore tag. I think this case fall into scope of OnlineCardPayment scope. I just moved it in this class to discuss further.